setup-buildengine.sh

```diff
@@ -584,8 +589,8 @@ startVM
 installTooling "$SAILFISH_DEFAULT_TOOLING"
 
 # install targets to the VM
-for targetname in $SAILFISH_DEFAULT_TARGETS; do
-    installTarget $targetname
+for targetarch in ${SAILFISH_DEFAULT_TARGET_ARCHES[*]}; do
+    installTarget "$OPT_TARGET_BASENAME-$arch-$OPT_RELEASE" ${OPT_TARGETS[$targetarch]}
```

Comment:
1) I believe the $arch should be $targetarch
2) I believe your parameters are reversed (installTarget expects filename first, target name second)
3) ... which makes me think OPT_TARGETS would be better called OPT_TARGET_FILENAMES